### ISSUE TITLE
Gadgetron needs boost-python

### ISSUE THREAD

**Issue by KrisThielemans** [CCPPETMR/SIRF-SuperBuild#15]
There seems to be a mistake in Gadgetron's CMake files such that it errors-out if boost-python is not found (as opposed to not building its Python support). So we need to compile this anyway at present.

**Comment by KrisThielemans** [comment]
we actually already build boost-python. However, we don't test for any boost components when using the system boost.

**Comment by KrisThielemans** [comment]
we now disable Gadgetron's Python support, so no longer need boost-python. @paskino can you check if we're still building it (note that the list of components to build are in 2 places, one for Windows and one for the rest)

**Comment by paskino** [comment]
Yes we are building it. For Unix is easy to remove the python module:
https://github.com/CCPPETMR/SIRF-SuperBuild/blob/master/SuperBuild/External_Boost_configureboost.cmake#L26
For Windows I don't quite understand where do we set the boost modules to build.

**Comment by KrisThielemans** [comment]
For Windows, it is in `External_Boost_buildboost.cmake.

**Comment by paskino** [comment]
All right, then for windows [isn't there](https://github.com/CCPPETMR/SIRF-SuperBuild/blob/master/SuperBuild/External_Boost_buildboost.cmake#L26) already.

**Comment by KrisThielemans** [comment]
actually, we have these variables already, e.g. [`BUILD_GADGETRON_NATIVE_PYTHON_SUPPORT`](https://github.com/CCPPETMR/SIRF-SuperBuild/blob/32831ceb546e6a7d61e9e50e6e728940a0758853/SuperBuild/External_Gadgetron.cmake#L56).
This means that we'd need to build `boost-python`, with lots of consequences, such as #56 and https://github.com/gadgetron/gadgetron/issues/550.
I suggest we disable `BUILD_GADGETRON_NATIVE_PYTHON_SUPPORT` for now, and then reassign this issue to v1.1 (or even later).

**Comment by KrisThielemans** [comment]
will disable this for v1.0.

**Comment by KrisThielemans** [comment]
it is disabled, so shifting milestone

**Comment by KrisThielemans** [comment]
removing from a milestone as nobody seems to want this.

**Comment by KrisThielemans** [comment]
we need to get away from Gadgetron, so closing.
